apps/api/src/applicant/entity/ienapplicant.entity.ts

```diff
@@ -35,7 +35,7 @@ export class IENApplicant {
   name!: string;
 
   @Index({ unique: true })
-  @Column('uuid', { nullable: true })
+  @Column('varchar', { nullable: true })
```

Comment:
Using the `integer` or `serial` type, we don't need to cast a number to string later.

Reply:
I normally like to keep things that don't require any mathematics as strings, varchar could be more flexible too.  My opinion, but I can change it too integer if we want, it may require more modifications to other files thinking it's still a `string`

Reply:
Is there any place referencing this field?
The `applicant_id` field points to `ien_applicants.id` in other tables. To avoid confusion, we could keep using the name `ats1_id`

Reply:
not in many places, I can change